runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useUserSearch.ts

```diff
@@ -28,7 +28,12 @@ const useUserSearch = (isActivatedUsersOnly: boolean, pageSize = 10, page = 0) =
     useEffect(() => {
         const newPage = search !== '' ? 0 : page;
 
-        router.replace({ pathname: router.pathname, query: { page: newPage, pageSize, search } });
+        if (search === '') {
+            router.replace({ pathname: router.pathname, query: { page: newPage, pageSize } });
+        } else {
+            router.replace({ pathname: router.pathname, query: { page: newPage, pageSize, search } });
+        }
```

Comment:
you can do it in a single-line
https://gist.github.com/preludium/6fa559527dc15f5d22c436e5b0df0b8b